.travis.yml

```diff
@@ -26,7 +26,7 @@ before_script:
 
 # command to run tests, e.g. python setup.py test
 script:
-  - tox -e $TOX_ENV
+  - tox -e $TOX_ENV -r
```

Comment:
Was this necessary? I initially wanted tox to reuse the first environment to generate our docs and run BDD.. not sure how well that played in the end.